#!/bin/bash

/usr/bin/python3.6 corebot.py > logs/errors.log
